lib/tiles.py

```diff
@@ -1,5 +1,5 @@
 import os
-from math import ceil, floor
+from math import ceil
```

Comment:
Oops, thanks.  What are you using for your git commit hook?  Clearly mine is missing stuff.

Reply:
Actually, you remind me to set up the hook again. I lost it when i cloned inkstitch again (for whatever reason) and now I do not have the commit hook anymore. I check manually with make style.

Reply:
I think I'm gonna have to change the hook script.  More recent versions of flake8 don't support `--diff` -- apparently it never worked perfectly and they decided to get rid of it since it wasn't possible to fix it.